dpdata/vasp/outcar.py

```diff
@@ -18,6 +18,14 @@ def system_info(lines, type_idx_zero=False):
                 atom_names.append(_ii.split("_")[0])
             else:
                 atom_names.append(_ii)
+        elif "POTCAR" in ii:
```

Comment:
my outcar reads
```
 INCAR:
 POTCAR:    PAW_PBE Mg 13Apr2007                  
 POTCAR:    PAW_PBE Mg 13Apr2007                  
   VRHFIN =Mg: s2p0                                                             
   LEXCH  = PE                                                                  
   EATOM  =    23.0369 eV,    1.6932 Ry                                         
                                                                                
   TITEL  = PAW_PBE Mg 13Apr2007                                                
   LULTRA =        F    use ultrasoft PP ?                                      
   IUNSCR =        1    unscreen: 0-lin 1-nonlin 2-no                           
```

Then the same atom name will be added three times.

Reply:
The dpdata is expected to abstract atom names in an order that is exact the same as how the potcars are concatenated. 
Your implementation does not work for the cases like "A" "B" "A" "B"